Parse-Dashboard/app.js

```diff
@@ -87,21 +87,29 @@ module.exports = function(config, options) {
         newFeaturesInLatestVersion: newFeaturesInLatestVersion,
       };
 
-      //Based on advice from Doug Wilson here:
-      //https://github.com/expressjs/express/issues/2518
-      const requestIsLocal =
-        req.connection.remoteAddress === '127.0.0.1' ||
-        req.connection.remoteAddress === '::ffff:127.0.0.1' ||
-        req.connection.remoteAddress === '::1';
-      if (!options.dev && !requestIsLocal) {
-        if (!req.secure && !options.allowInsecureHTTP) {
+      for (const key in options) {
+        if (options[key] && !config[key]) {
+          config[key] = options[key];
+        }
+      }
+      if (!config.dev) {
+        if (!req.secure && !config.allowInsecureHTTP) {
           //Disallow HTTP requests except on localhost, to prevent the master key from being transmitted in cleartext
-          return res.send({ success: false, error: 'Parse Dashboard can only be remotely accessed via HTTPS' });
+          return res.send({
+            success: false,
+            error:
+              "Parse Dashboard can only be remotely accessed via HTTPS.",
+            log: "Parse Dashboard can only be remotely accessed via HTTPS. If you are running locally, use the --dev parameter to bypass allowInsecureHTTP.",
```

Comment:
I think "bypass" is ambiguous, it means "undefined state" in this case because it doesn't indicate whether the `allowInsecureHTTP` is true or false when it's "bypassed".

That wording is also used elsewhere in the docs. Maybe we can be more clear and say it "overrides the option to `false`" or something like that?